Asterix/Example_param_file.ini

```diff
@@ -26,7 +26,8 @@ onbench=True
 
     # Sampling in the detector science image
     #(lambda/Entrance_pupil_diameter in pixel
-    Science_sampling = 7.25
+    # For THD2 testbed : 7.42 pix / L/D for the 783.6 laser
+    Science_sampling = 7.42
```

Comment:
Are you sure this really refers to the entrance pupil diameter? If this was measured on testbed data, the sampling of 7.42 pixels per lam/D is actually in Lyot stop diameter. We should absolutely clarify this.